src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,25 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        if (names.length == 0 || data.length == 0 || dateFrom.isEmpty() || dateTo.isEmpty()) {
+            return null;
```

Comment:
Returning null in case of empty input parameters is not a good practice. It would be better to return an empty string or throw an IllegalArgumentException with a descriptive message.